packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -396,6 +420,24 @@ class _DraggableState<T> extends State<Draggable<T>> {
   }
 }
 
+/// Represents the draggable's current details, such as whether the
+/// [DragTarget] accepted it, its [Velocity], and its [Offset].
+class DraggableDetails {
+  /// Determines whether the [DragTarget] accepted this draggable
+  final bool wasAccepted;
+
+  /// This draggable's current [Velocity]
+  final Velocity velocity;
+
+  /// This draggable's current [Offset]
+  final Offset offset;
+
+  DraggableDetails._({
```

Comment:
Constructors come first in the class declaration, according to Flutter style.

Reply:
Will make that change very shortly.